### PR TITLE
Rebranding and qt wallet compilation

### PR DESCRIPTION
**Pull request by yghazi**
QT wallet compiles on linux and it's working.
Pretty much done with the wallet rebranding, save for zmq, which I haven't enabled yet.
Also, need logos and images, of course, but not exactly a priority right now.
Need to remove the translation files. Just renamed them right now.

If anyone could check if the rebranded wallet compiles on Windows and Mac, that'd be great.

### CODE REVIEW

File: src/qt/utilitydialog.cpp [hunk 1]
```diff
@@ -135,13 +135,13 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, HelpMode helpMode) :
...
 PrivateSend gives you true financial privacy by obscuring the origins of your funds. \
-All the Dash in your wallet is comprised of different \"inputs\" which you can think of as separate, discrete coins.<br> \
+All the Energi in your wallet is comprised of different \"inputs\" which you can think of as separate, discrete coins.<br> \
 PrivateSend uses an innovative process to mix your inputs with the inputs of two other people, without having your coins ever leave your wallet. \
 You retain control of your money at all times..<hr> \
 <b>The PrivateSend process works like this:</b>\
 <ol type=\"1\"> \
 <li>PrivateSend begins by breaking your transaction inputs down into standard denominations. \
-These denominations are 0.01 DASH, 0.1 DASH, 1 DASH and 10 DASH -- sort of like the paper money you use every day.</li> \
+These denominations are 0.01 ENERGI, 0.1 ENERGI, 1 ENERGI and 10 ENERGI -- sort of like the paper money you use every day.</li> \
```
Comment (ediston) [inline src/qt/utilitydialog.cpp]: to be consistent EGI instead of ENERGI

File: src/qt/walletmodel.cpp [hunk 2]
```diff
@@ -305,7 +305,7 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
         CReserveKey *keyChange = transaction.getPossibleKeyChange();
 
         if(recipients[0].fUseInstantSend && total > sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)*COIN){
-            Q_EMIT message(tr("Send Coins"), tr("InstantSend doesn't support sending values that high yet. Transactions are currently limited to %1 DASH.").arg(sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)),
+            Q_EMIT message(tr("Send Coins"), tr("InstantSend doesn't support sending values that high yet. Transactions are currently limited to %1 ENERGI.").arg(sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)),
```
Comment (ediston) [inline src/qt/walletmodel.cpp]: EGI

File: src/qt/walletmodel.cpp [hunk 3]
```diff
@@ -317,7 +317,7 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
 
         if(recipients[0].fUseInstantSend) {
             if(newTx->GetValueOut() > sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)*COIN) {
-                Q_EMIT message(tr("Send Coins"), tr("InstantSend doesn't support sending values that high yet. Transactions are currently limited to %1 DASH.").arg(sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)),
+                Q_EMIT message(tr("Send Coins"), tr("InstantSend doesn't support sending values that high yet. Transactions are currently limited to %1 ENERGI.").arg(sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)),
```
Comment (ediston) [inline src/qt/walletmodel.cpp]: EGI

**Review by jmgilbert2 — COMMENTED** [review]
Still some cases of random Energi use for the coin unit. Also spotted "a Energi address". Sorry if all comments in one batch, haven't yet figured out how to comment on individual files in GitHub.

**Comment by ediston** [comment]
Why 'energi-config.h' is being deleted ?

**Comment by yghazi** [comment]
> Why 'energi-config.h' is being deleted ?
Because it's automatically generated, and I forgot to include it in the .gitignore the first time around.

**Pull request merged**